views/default/core/account/login_dropdown.php

```diff
@@ -12,18 +12,17 @@
 ], [
 	'returntoreferer' => true,
 ]);
-?>
-<div id="login-dropdown">
-	<?php
-		echo elgg_view('output/url', [
-			'href' => elgg_get_login_url([], '#login-dropdown-box'),
-			'rel' => 'popup',
-			'text' => elgg_echo('login'),
-			'data-position' => json_encode([
-				'my' => 'right top',
-				'at' => 'right bottom',
-			]),
-		]);
-		echo elgg_view_module('dropdown', '', $body, ['id' => 'login-dropdown-box']);
-	?>
-</div>
+
+$body = elgg_view('output/url', [
+	'href' => elgg_get_login_url([], '#login-dropdown-box'),
+	'rel' => 'popup',
+	'text' => elgg_echo('login'),
+	'data-position' => json_encode([
+		'my' => 'right top',
+		'at' => 'right bottom',
+	]),
+]);
+
+$result = elgg_view_module('dropdown', '', $body, ['id' => 'login-dropdown-box']);
```

Comment:
this is wrong. The URL was before the module so it could be toggled. Now it's inside. Also maybe `$body` is completely overruled by the url.

Reply:
good catch... fixed it